packages/core/package.json

```diff
@@ -0,0 +1,72 @@
+{
+	"name": "@discordjs/core",
+	"version": "0.1.0",
+	"description": "",
+	"scripts": {
+		"test": "vitest run",
+		"build": "tsup",
+		"lint": "prettier --check . && cross-env TIMING=1 eslint src --ext mjs,js,ts",
```

Comment:
add changes from https://github.com/discordjs/discord.js/commit/e8f63617a231c028591a5cdd008bd0bb5f5f43cb and add `eslint-formatter-pretty` deps